plover/gui_qt/add_translation.py

```diff
@@ -153,12 +151,61 @@ def _translation(self):
         translation = self.translation.text().strip()
         return unescape_translation(translation)
 
+    def _update_items(self, dictionaries=None, reverse_order=None):
+        if dictionaries is not None:
+            self._dictionaries = dictionaries
+        if reverse_order is not None:
+            self._reverse_order = reverse_order
+        iterable = self._dictionaries
+        if self._reverse_order:
+            iterable = reversed(iterable)
+        self.dictionary.clear()
+        for d in iterable:
+            item = shorten_path(d.get_path())
+            if not d.enabled:
+                item += ' [' + _('disabled') + ']'
```

Comment:
I'm not sure I'm seeing this change...should it auto select the first enabled dictionary and label disabled ones?

With no selection and my first two dictionaries disabled I get this result

<img width="504" alt="screen shot 2017-04-01 at 12 36 20 am" src="https://cloud.githubusercontent.com/assets/5840970/24575430/51976e5e-1673-11e7-81d9-fce902c9a032.png">